src/keyring.ts

```diff
@@ -0,0 +1,250 @@
+import type { Transaction } from '@ethereumjs/tx';
```

Comment:
TODO: Find a replacement `Transaction` type that doesn't tie us to this specific library.

Reply:
Maybe this is what we need: https://github.com/ethereumjs/ethereumjs-monorepo/blob/7d7b0ad0915b560093a7e1ac5802432e40f436f5/packages/tx/src/types.ts#L143